### DISCUSSION TITLE
UnauthorizedError: The maximum number of teams has been reached

### DISCUSSION THREAD

**Discussion by oakbrad** [outline/outline#4961] [Self Hosting]
New invited user unable to login with Google because "maximum number of teams has been reached". 

`{"error":"The maximum number of teams has been reached","stack":"UnauthorizedError: The maximum number of teams has been reached\n    at InvalidAuthenticationError (/opt/outline/build/server/errors.js:44:34)\n    at accountProvisioner (/opt/outline/build/server/commands/accountProvisioner.js:56:52)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","level":"error","message":"Error during authentication"}`

I previously have had no issues inviting users or having them login with their Google accounts.

**To Reproduce**
1. Invite user@gmail.com to Outline
2. User tries to log in with Google, generates below error
3. Magic sign in link works fine 

**Expected behavior**
Should be able to login with Google and not just magic link

**Outline (please complete the following information):**
- Install: self hosted (Docker)
- Version: 0.68.1 (commit `5c6475f`)

**Comment by tommoor** [comment]
It sounds like they are attempting to sign-in with a Google account that isn't authorized to access the workspace. You can invite them first, or add the domain as an authorized domain.

**Comment by dabaobbb** [comment]
I used gitlab to login ,show the same error log
